ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -183,12 +190,19 @@ def install_lso_cluster(self):
             logger.error("Nodes not found after 600 seconds")
             raise TimeoutExpiredError
 
-        logger.info(f"Select {constants.LOCAL_BLOCK_RESOURCE} storage class")
-        self.choose_expanded_mode(
-            mode=True, locator=self.dep_loc["storage_class_dropdown_lso"]
+        if self.operator == OCS_OPERATOR:
+            logger.info(f"Select {constants.LOCAL_BLOCK_RESOURCE} storage class")
+            self.choose_expanded_mode(
+                mode=True, locator=self.dep_loc["storage_class_dropdown_lso"]
+            )
+            self.do_click(locator=self.dep_loc["localblock_sc"], enable_screenshot=True)
+            timeout_next = 30
+        else:
+            timeout_next = 600
+
+        self.do_click(
+            self.dep_loc["next"], enable_screenshot=True, timeout=timeout_next
```

Comment:
would that mean we are waiting for 10 minutes instead of 30 seconds?

Reply:
Because it takes time for "Available raw capacity" to be updated
http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/jnk-pr5061-b1815/jnk-pr5061-b1815_20211110T230314/logs/screenshots_ui_1636585677/test_deployment/2021-11-10T23-35-26.816813.png